charts/prometheus/values.yaml

```diff
@@ -1297,7 +1297,7 @@ serverFiles:
       - job_name: prometheus
         static_configs:
           - targets:
-            - localhost:9090
+            - localhost:9090{{.Values.server.prefixURL}}/metrics
```

Comment:
this looks like it breaks it for updating users

Reply:
I was thinking about this, but in which case would it break?

1. The user did not specify a prefixURL (the default) -> nothing changes
2. The user has already specified a prefixURL (e.g. because they are using a baseURL with a path) -> it was already broken for them, since Prometheus was listening on `/prefix/metrics`, but the scrape target did not reflect this

Am I missing something?

Reply:
The default without prefix url would change from localhost:9090 to localhost:9090/metrics

Reply:
In fact, we should configure `prefixURL/metrics` on `metrics_path` not `static_config` according to https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config

Reply:
@naseemkullah Excellent suggestion! I have updated the PR accordingly.